src/ast/FuncDecl.ts

```diff
@@ -9,26 +9,93 @@ import {Tokenizer} from "../util/Tokenizer";
  * e.g. function modifier (async | static)? str
```

Comment:
It should be `function modifier 'async '? 'static'? str`

Reply:
Yup, you're right. I think it should be 

`function mod (static)? (async)? str`

since TypeScript won't compile with the type signature

`public async static ...`